src/Analyzers/MSTest.Analyzers.CodeFixes/Helpers/Utils.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Analyzer.Utilities;
+
+using Microsoft.CodeAnalysis;
+
+namespace MSTest.Analyzers.Helpers;
+
+internal class Utils
+{
+    public static bool IsDiscoverInternalsAttributePresent(SemanticModel semanticModel)
```

Comment:
This would be better as some extension method on the Compilation, maybe renamed as `HasDiscoverInternalAttribute`